liquidhaskell.cabal

```diff
@@ -135,7 +135,7 @@ Library
                 , vector >= 0.10
                 , hashable >= 1.2
                 , unordered-containers >= 0.2
-                , liquid-fixpoint >= 0.5 && < 0.6
+                , liquid-fixpoint 
```

Comment:
Feel free to fiddle and fix with the bounds -- I removed it as the thing wasn't building ...

> On Apr 19, 2016, at 6:37 PM, Eric Seidel <notifications@github.com> wrote:
> 
> In liquidhaskell.cabal:
> 
> > @@ -135,7 +135,7 @@ Library
> >                  , vector >= 0.10
> >                  , hashable >= 1.2
> >                  , unordered-containers >= 0.2
> > -                , liquid-fixpoint >= 0.5 && < 0.6
> > +                , liquid-fixpoint 
> Please leave this bound in place, liquid-fixpoint changes API so much it's pretty much guaranteed the next hackage release won't work with current LH.
> 
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly or view it on GitHub
>